bin/cli.js

```diff
@@ -76,7 +76,9 @@ localesPaths.forEach(localesPath => {
     const sourceLocale = program.sourceLocale || source || globalSource;
     const allowedLocalesString = program.locales || configLocales || globalLocales;
     const allowedLocales = allowedLocalesString && allowedLocalesString.split(',');
-    const filteredFiles = !allowedLocales ? files : files.filter(file => allowedLocales.includes(file.replace(`.${ext}`, '')) || file === sourceLocale + `.${ext}`);
+    const filteredFiles = !allowedLocales ?
+      files.filter(file => !(/^\..*/g).test(file)) :
```

Comment:
Don't include anything that starts with a `.`.

Reply:
Wow, I didn't realize you could use and test against a regex this way. Neat!